docs/supported-devices.md

```
@@ -125,6 +123,13 @@ entity in HA will be capped.
 You can still connect with the EVO app via Bluetooth after setting the OCPP server,
 but certain features (eg. scheduling) may not work.
 
+## [Schneider Electric EVlink ProAC](https://www.se.com/uk/en/product-range/23107242-evlink-pro-ac/)
```

markdownlint-cli2 (0.23.1)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)